src/sql/workbench/contrib/notebook/browser/htmlMarkdownConverter.ts

```diff
@@ -295,3 +307,16 @@ export function addHighlightIfYellowBgExists(node, content: string): string {
 	}
 	return content;
 }
+
+export function getUriAnchorLink(node, notebookUri: URI): URI {
```

Comment:
node should have a type

Reply:
Looks like we aren't currently importing types for turndown - it looks like DefinitelyTyped has them though so it would be great if you could look into adding these since right now it's a lot of guessing and assumptions around the properties on these things. 

https://www.npmjs.com/package/@types/turndown

(this can be done as a separate PR)